flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/plan/util/ExpandUtil.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.plan.util
+
+import org.apache.calcite.rel.`type`.RelDataType
+import org.apache.calcite.rex.{RexInputRef, RexLiteral, RexNode}
+
+import java.util
+
+import scala.collection.JavaConversions._
+
+/**
+  * Utility methods for expand operator.
+  */
+object ExpandUtil {
+
+  def projectsToString(
```

Comment:
Can we try to move all methods like these to one util class? Various logical or physical operators will share some name formatting utilities. There will be a lot repetition if we organize it by operator.